src/util/escapeCircumflexOnWindows.js

```diff
@@ -0,0 +1,13 @@
+const isWindows = require('./isWindows');
+
+/**
+ * Escape the circumflex character when we are on Windows
+ * since Batch will interpret it.
+ *
+ * @param {string[]} strings
+ */
+function escapeCircumflexOnWindows(strings) {
```

Comment:
```suggestion
function escapeArguments(strings) {
```

Otherwise we're leaking an implementation detail.

Reply:
Do you want me to change the function's name to `escapeArguments`?

Reply:
Yes, because not it escapes arguments on all platforms and the Windows check is an implementation detail, and we shouldn't leak implementation details.